docs/manual/interpreters.md

```diff
@@ -60,3 +60,16 @@ Each Interpreter Setting can choose one of 'shared', 'scoped', 'isolated' interp
 In 'shared' mode, every notebook bound to the Interpreter Setting will share the single Interpreter instance. In 'scoped' mode, each notebook will create new Interpreter instance in the same interpreter process. In 'isolated' mode, each notebook will create new Interpreter process.
 
 <img src="../assets/themes/zeppelin/img/screenshots/interpreter_persession.png" width="400px">
+
+
+## Connecting to the Existing Remote Interpreter
+
+Zeppelin users can start interpreter thread embedded in their service. This will provide flexibility to user to start interpreter on remote host. To start interpreter along with your service you have to create an instance of ``RemoteInterpreterServer`` and start it as follows:
+
+``RemoteInterpreterServer interpreter=new RemoteInterpreterServer(3789); ``  
```

Comment:
@SachinJanani 
Currently this part is shown like below 
<img width="962" alt="screen shot 2016-06-09 at 11 28 25 pm" src="https://cloud.githubusercontent.com/assets/10060731/15955965/0a90bbbc-2e9a-11e6-82cc-7a3bc779640c.png">

How about changing these 3 code block to like this ?
<img width="525" alt="screen shot 2016-06-09 at 11 33 33 pm" src="https://cloud.githubusercontent.com/assets/10060731/15956057/b4d5edc2-2e9a-11e6-81ab-282b5d9bf681.png">


Then it will be like this 
<img width="972" alt="screen shot 2016-06-09 at 11 28 07 pm" src="https://cloud.githubusercontent.com/assets/10060731/15955981/1c53571a-2e9a-11e6-9823-5d04223794c4.png">